x-pack/test/functional/page_objects/search_navigation.ts

```diff
@@ -17,6 +17,14 @@ export function SearchNavigationProvider({ getService, getPageObjects }: FtrProv
   const testSubjects = getService('testSubjects');
 
   return {
+    async navigateToElasticsearchOverviewPage(basePath?: string) {
+      await retry.tryForTime(60 * 1000, async () => {
+        await common.navigateToApp('elasticsearch/overview', {
```

Comment:
```suggestion
        await common.navigateToApp('enterpriseSearch', {
```

the apps you can navigate to are included in the functional config. `enterpriseSearch` [is defined here](https://github.com/elastic/kibana/blob/d6522194b87e2f078786744d8a1b93663d7ddb63/x-pack/test/functional/config.base.js#L193)